src/components/FieldWrapper/FieldWrapper.tsx

```diff
@@ -8,6 +9,10 @@ const { block, elem } = bem('FieldWrapper', styles);
 interface Props extends React.HTMLAttributes<HTMLDivElement> {
     /** content of the wrapper */
     children: NotEmptyReactNode;
+    /** show dropdown icon */
+    showArrow?: boolean;
+    /** set direction of dropdown icon (default drop down) */
+    isArrowDropUp?: boolean;
```

Comment:
I find the `drop` confusing. Can it be just `isArrowUp`?

Reply:
Same, I'd propose `isArrowPointingUp` or `isArrowUp`

Reply:
`isArrowUp` sounds better